packages/default-theme/components/SwProductDetails.vue

```diff
@@ -10,9 +10,11 @@
         :price="getPrice | price"
       />
     </div>
+    <SwPluginSlot name="product-page-description-before" />
     <p class="product-details__description desktop-only">
       {{ description }}
```

Comment:
Similar thing here - let's say you have a custom description field you'd like to display. How would you do that (without using CSS of course).

Reply:
https://github.com/DivanteLtd/shopware-pwa/pull/695/commits/b18fb19978927564ad15854dab22dbc2dd3e5303#diff-92fbafed2edd6afbfc8768ae9dbf3bf9 done as well :)